trino/trino.go

```diff
@@ -114,6 +121,19 @@ const (
 	SSLCertPathConfig        = "SSLCertPath"
 )
 
+var (
+	headersMap = map[string]string{
+		trinoSetSchemaHeader:  trinoSchemaHeader,
+		trinoSetCatalogHeader: trinoCatalogHeader,
+	}
+	unsupportedHeaders = []string{
```

Comment:
nit: rename to `unsupportedResponseHeaders`

Reply:
Done!